src/groups/semidirect_product_group.jl

```diff
@@ -173,6 +173,14 @@ function translate_diff!(G::SemidirectProductGroup, Y, p, q, X, conX::LeftAction
     return Y
 end
 
+# We need to prevent decorator unwrapping so that the correct `get_vector!` gets called
+# and applies proper padding to the result if `X` happens to be a matrix.
+# Otherwise rare random bugs happen where the padding is not applied.
+function get_vector(G::SemidirectProductGroup, p, X, B::VeeOrthogonalBasis)
+    Y = allocate_result(G, get_vector, p, X)
+    return get_vector!(G, Y, p, X, B)
+end
+
```

Comment:
Can this also be done setting the transparency right? I think so (if this is too complicated, overwriting it like here is of course also one way to go.

Reply:
It's one of those cases where writing a short method manually is much easier than figuring out transparency.

Reply:
Ok.